scholia/config.py

```diff
@@ -0,0 +1,46 @@
+"""Configuration for Scholia."""
+from __future__ import absolute_import
+
+try:
+    import ConfigParser as configparser
+except ImportError:
+    import configparser
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+import logging
+
+from os.path import exists, expanduser
+
+
+CONFIG_FILENAMES = [
+    'scholia.cfg',
+    '~/etc/scholia.cfg',
+    '~/scholia.cfg']
+
+DEFAULTS = """
+[servers]
+SPARQLEndPointURLbd = https://query.wikidata.org/bigdata/namespace/wdq/sparql
```

Comment:
Is this variable necessary?

Reply:
That's a good question. 
This URL is used in a couple of places in scholia. 
If it makes sense to use a bigdata separate endpoint than it makes sense to make it into a configuration also.

Note:
This could also be done differently, by having a single variable for the SPARQL endpoint and appending the suffix in the code, depending on if you want to access bigdata or just the normal endpoint. My choice keeps more flexibility since we can use completely different URLs for bigdata and normal queries.

Reply:
I think that `https://query.wikidata.org/bigdata/namespace/wdq/sparql` is just an old endpoint and we should have changed that long ago. Are there really difference between that and /sparql endpoint?

Reply:
I have no idea. I thought that had some special purpose.
I will remove that configuration and set the URL to the main one in scholia if you agree.

Reply:
I have now asked https://www.wikidata.org/w/index.php?title=Wikidata_talk:SPARQL_query_service&diff=1111375482&oldid=1053274607

Reply:
@fnielsen do you think you can accept this pull request and, depending on the answer, do further adaptations?
I have a couple of other pull requests to commit and request. I don't want to do that before closing this one though.